risc0/cargo-risczero/src/commands/new.rs

```diff
@@ -116,16 +116,20 @@ impl NewCommand {
                 format!("git = \"https://github.com/risc0/risc0.git\", branch = \"{branch}\"");
             template_variables.push(format!("risc0_build={spec}"));
             template_variables.push(format!("risc0_zkvm={spec}"));
+            template_variables.push(format!("risc0_zkvm_platform={spec}"));
```

Comment:
Doesn't `risc0-zkvm` depend on `risc0-zkvm-platform` already?

Reply:
This is vestigial. Addressed in c75f8b613b940896a4ae5be82eedfbad9a999b7f.